### ISSUE TITLE
Request for pretrained weights and clarification on inference pipeline

### ISSUE THREAD

**Issue by Gao-anqi** [Guo777777/CBCT-mandanbular-and-maxillary-Landmark-detection#1]
Hello,

Thank you very much for sharing the code for your work, *“Cone-beam CT landmark detection for measuring basal bone width: a retrospective validation study.”* I am very interested in applying and externally validating this method on our own CBCT dataset.

I have been trying to reproduce the inference pipeline using the current repository, but I encountered several missing files/checkpoints and would greatly appreciate your clarification.

From the current code, I noticed:

1. In `tooth_region_pred.py`, the model is imported as:

   ```python
   from UNet_xhy import UNet3D_simple
   ```

   and the script loads:

   ```text
   tooth_best_model/bestmodel.pth
   ```

   However, I could not find `UNet_xhy.py` or `bestmodel.pth` in the repository.

2. In `2_train_and_valid.py` and `3_test.py`, the landmark model is imported as:

   ```python
   from UNet import UNet3d
   ```

   while the repository currently contains `UNet_global.py` and `UNet_local.py`, but not `UNet.py`.

   Could you please confirm whether:

   * `UNet_xhy.py` corresponds to the current `UNet_global.py`, and
   * `UNet.py` corresponds to the current `UNet_local.py`?

3. `3_test.py` loads the checkpoint:

   ```text
   UNet3d_stage1/Unet_model08.pt
   ```

   but this pretrained checkpoint also does not appear to be publicly available.

4. I was also wondering about the landmark heatmap implementation. The current training/test code uses:

   ```python
   UNet3d(n_class=1)
   ```

   whereas the paper describes landmark heatmap regression for four landmarks. Is the final model based on a single heatmap containing four landmark regions, or four separate output channels?

Would it be possible to provide or clarify:

* The pretrained `bestmodel.pth` for tooth-region localization;
* The pretrained `Unet_model08.pt` (and any other checkpoints required for the complete two-stage model);
* The missing model definition files, or confirmation that `UNet_global.py` / `UNet_local.py` are the corresponding implementations;
* The preprocessing requirements for a new CBCT case, including orientation, voxel spacing/resampling, normalization, and expected input sizes;
* If available, a minimal end-to-end inference script from a new CBCT/NIfTI volume to the four landmark coordinates and final maxillary/mandibular basal bone width measurements.

Our intended use is purely academic research and external validation, and we will of course properly cite your paper and repository in any resulting work.

If the pretrained weights cannot be released publicly, I would also be very grateful if they could be shared privately for research use, if possible.

Thank you very much for your time and for making the code available.
